src/Avalonia.Controls/Screens.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Linq;
+using Avalonia.Platform;
+using Avalonia.Utilities;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Controls
+{
+    public class Screens
+    {
+        private readonly IScreenImpl _iScreenImpl;
+
+        public int ScreenCount => _iScreenImpl.ScreenCount;
+        public Screen[] All => _iScreenImpl?.AllScreens;
+        public Screen Primary
+        {
```

Comment:
You lambda the previous two properties, why not this one too?

Reply:
That used to be a for loop, I'll change it.